app/src/main/res/navigation/mobile_navigation.xml

```diff
@@ -14,7 +14,7 @@
     <fragment
         android:id="@+id/navigation_dashboard"
         android:name="com.example.habitsmasher.ui.dashboard.DashboardFragment"
-        android:label="@string/title_dashboard"
+        android:label="@string/habit_list"
```

Comment:
This changes the menu bar string:
![image](https://user-images.githubusercontent.com/43623225/136719466-baae9ae4-ddd3-4aef-be1c-8e121aa21e86.png)